.gitignore

```diff
@@ -1,29 +1,10 @@
-# Ignore directories generated by Composer
-/.editorconfig
-/.gitattributes
-/vendor/
-/web/_ping.php
-/web/.gitignore
-/web/core/
-/web/modules/.gitignore
-/web/modules/contrib/
-/web/themes/.gitignore
-/web/themes/contrib/
-/web/profiles/.gitignore
-/web/profiles/contrib/
-/web/libraries/
-/web/sites/.gitignore
-/web/sites/default/.gitignore
+# Ignore .gitattributes files
+.gitattributes
+.editorconfig
 
-# Ignore local configuration
-/web/sites/*/settings.local.php
-
-# Ignore Drupal's file directory
-/web/sites/*/files/
-
-# Ignore testing directories.
-/web/sites/simpletest
-/web/tmp
+# Ignore potential database dumps:
+*.sql.*
+*.sql
 
 # Ignore files generated by PhpStorm
 /.idea/
```

Comment:
Remove leading /

Reply:
Done in https://github.com/wunderio/next4drupal-project/pull/10/commits/9b36fbdd4fbb64fc73d22620eca857d4abd75adc